docs/source/usage.rst

```diff
@@ -0,0 +1,54 @@
+Usage
+#####
+
+Basic usage with raster files
+=============================
+
+
+
+.. code-block:: python
```

Comment:
Too much whitespace here. Let's just have a single blank line between the heading and the start of the code block.

```suggestion
=============================

.. code-block:: python
```